src/ClientModel/CaptureData.cpp

```diff
@@ -111,6 +111,37 @@ void CaptureData::UpdateFunctionStats(uint64_t instrumented_function_id, uint64_
   }
 }
 
+void CaptureData::OnCaptureComplete(std::vector<orbit_client_data::TimerChain*> chains) {
+  // Recalculate standard deviation as the running calculation may have introduced error.
+  for (auto& pair : functions_stats_) {
+    FunctionStats& stats = pair.second;
+    stats.set_variance_ns(0);
+  }
+
+  for (auto& chain : chains) {
+    if (!chain) continue;
+    for (const auto& block : *chain) {
+      for (uint64_t i = 0; i < block.size(); i++) {
+        const orbit_client_protos::TimerInfo& timer_info = block[i];
+        auto& stats = functions_stats_[timer_info.function_id()];
```

Comment:
This will insert an entry into the hash_map if the function_id wasn't a key already. Maybe use `find()` instead of `[]` and continue if the key wasn't present?

Also, for simple types, we shouldn't use auto.

Reply:
done